docs/contributing/CHANGELOG_TEMPLATE.md

```diff
@@ -2,28 +2,23 @@
 
 ## Unreleased
 
-💥 Breaking changes:
+Breaking changes
 
-- Pull Request Title goes here
+#### Heading (h4) to summarise change goes here
 
-  Description goes here (optional)
+Description goes here
 
-  To migrate you need to change: X
+This change was introduced in [pull request #xxxx: PULL-REQUEST-TITLE](LINK-TO-PULL-REQUEST). (optional) Thanks to [@GITHUB-NAME](LINK-TO-CONTRIBUTOR-GITHUB-PAGE) for reporting/contributing this issue.
 
-  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
+New features
```

Comment:
Same comment here about marking it up as a h3

Reply:
```suggestion
### New features
```